src/wp-includes/class-wp-script-modules.php

```diff
@@ -343,6 +345,20 @@ public function print_enqueued_script_modules() {
 			if ( $fetchpriority !== $script_module['fetchpriority'] ) {
 				$args['data-wp-fetchpriority'] = $script_module['fetchpriority'];
 			}
+
+			/**
+			 * Filters the attributes for a script module tag.
+			 *
+			 * @since 6.9.0
+			 *
+			 * @param array  $args          Key-value pairs representing `<script>` tag attributes.
+			 *                              Only the attribute name is added to the `<script>` tag for
+			 *                              entries with a boolean value, and that are true.
+			 * @param string $id            The script module identifier.
+			 * @param array  $script_module The script module data.
+			 */
+			$args = apply_filters( 'wp_script_module_attributes', $args, $id, $script_module );
```

Comment:
I'd make the arguments match the existing filter for scripts ([`wp_script_attributes`](https://developer.wordpress.org/reference/hooks/wp_script_attributes/)):
 
- Rename `$args` to `$attributes`
- Remove `$id`, which can be obtained from the `$attributes['id']` anyway
- Eliminate `$script_module`, whose schema is not publicly exposed for now, and this exposure would mean that backward compatibility would have to be maintained from now on.


```suggestion
			$args = apply_filters( 'wp_script_module_attributes', $attributes );
```